config/prod/tom-cvat.yaml

```diff
@@ -0,0 +1,39 @@
+template_path: remote/managed-ec2.yaml
+stack_name: tom-cvat
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "CompOnc"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "dream"
+  # The resource owner
+  OwnerEmail: "thomas.yu@sagebase.org"
+
+  VpcName: "{{stack_group_config.default_vpc}}"
+  KeyName: "{{stack_group_config.default_key_pair}}"
+  AMIId: "{{stack_group_config.default_awslinux2_ami}}"
+  
+  # The following parameters are only examples they are not required.
+  # You may omit them if you do not need to override the defaults.
+
+  # (Optional) EC2 instance type, default is "t2.nano" (other available types https://aws.amazon.com/ec2/instance-types/)
+  InstanceType: "t2.xlarge"
+  # (Optional) EC2 boot volume size in GB, default is 8GB, Max is 1000GB
+  VolumeSize: "500"
+  # (Optional) Run EC2 in a subnet (default: PrivateSubnet) (valid values: PrivateSubnet or PublicSubnet)
+  VpcSubnet: "PublicSubnet"
+  # (Optional) Expose a port to incoming traffic (default is no open ports) (valid range: 1-65535)
+  OpenPort: "80,8082"
```

Comment:
multiple ports isn't supported, or not yet anyways.   I think the workaround for now is to open one port for the instance then manually add a second port after the instance has been provisioned.

Reply:
Got it.  Thats what I did the first time around